packages/cli/src/databases/repositories/folder.repository.ts

```diff
@@ -8,21 +8,34 @@ import { Folder } from '../entities/folder';
 import { FolderTagMapping } from '../entities/folder-tag-mapping';
 import { TagEntity } from '../entities/tag-entity';
 
+export const isFolder = (item: unknown): item is Folder =>
+	typeof item === 'object' && item !== null && 'resource' in item && item.resource === 'folder';
```

Comment:
I think this could be moved to `workflow.service` since it's only used there. Also the param could be typed a `WorkflowAndFolderUnionFull` to simplify the check

Reply:
removed it altogether